src/cozmo/robot.py

```diff
@@ -392,6 +392,25 @@ def _encode(self):
             faceID=self.face.face_id)
 
 
+
+class PerformOffChargerContext(event.Dispatcher):
+    '''A helper class to provide a context manager to do operations while Cozmo is off charger.'''
+    def __init__(self, robot, **kw):
+        super().__init__(**kw)
+        self.robot = robot
+
+    async def __aenter__(self):
+        self.was_on_charger = self.robot.is_on_charger
+        self.was_on_charger = True
```

Comment:
ugh no.. it's because is_on_charger is returning False for my setup for some reason, and not for Michelle's.. will remove.